test/integration/authorization_test.go

```diff
@@ -488,8 +488,6 @@ func TestAuthorizationResourceAccessReview(t *testing.T) {
 		}
 		test.response.Users.Insert(globalClusterReaderUsers.List()...)
 		test.response.Users.Insert(globalDeploymentConfigGetterUsers.List()...)
-		test.response.Users.Delete("system:serviceaccount:openshift-infra:template-instance-controller")
-		test.response.Users.Delete("system:serviceaccount:openshift-infra:template-instance-finalizer-controller")
```

Comment:
An aside - why did this test delete the template instance controller SAs in the first place?

Reply:
Before this change @adambkaplan the templat controller SAs did *NOT* have cluster-admin capabilities (unlike the other users in the test response list), so they had to be removed so the compare against the list of users with cluster-admin like capabilities matched up.

Since we are giving them now the power to perform such things, we no longer remove them from the list to verify against.

Reply:
seems slightly odd the test didn't just expect the TIC SAs to show up in the list.  oh well.